### PR TITLE
BI-8472 Allocate higher RAM for the data-reconciliation app

### PR DESCRIPTION
**Pull request by kinpang-CH**
* Bump memory allocation from 16GB to 30GB on all profiles
* Designate 30minutes to cache expiry time instead of 5
* Set the max heap size for java applications to 16GB

### CODE REVIEW

**Review by cgregoryCH — DISMISSED** [review]

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
Can you leave the memory variable as-is, please? Thanks

File: groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -5,7 +5,7 @@ deploy_to = "development"
 state_prefix = "env:/development"
 aws_profile = "development-eu-west-2"
 cpu_units = 1024
-memory = 8192
+memory = 30720
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars]: Please revert this change.

File: groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars [hunk 2]
```diff
@@ -36,3 +36,5 @@ insolvency_case_count_mongo_oracle_enabled = true
 
 results_initial_capacity = 1000000
 results_expiry_time_in_millis = 1800000 # 30 minutes
+cache_expiry_in_seconds = 1800
+java_mem_args = "-Xmx14336m"
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars]: Should be:

```
java_mem_args = "-Xmx7g"
```

File: groups/data-reconciliation/profiles/live-eu-west-2/live/vars [hunk 3]
```diff
@@ -5,7 +5,7 @@ deploy_to = "live"
 state_prefix = "env:/live"
 aws_profile = "live-eu-west-2"
 cpu_units = 2048
-memory = 16384
+memory = 30720
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/live-eu-west-2/live/vars]: Please revert this change.

File: groups/data-reconciliation/profiles/live-eu-west-2/live/vars [hunk 4]
```diff
@@ -38,3 +38,5 @@ mongodb_read_preference = "SECONDARY"
 
 results_initial_capacity = 1000000
 results_expiry_time_in_millis = 86400000 # 1 day
+cache_expiry_in_seconds = 1800
+java_mem_args = "-Xmx14336m"
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/live-eu-west-2/live/vars]: Should be:

```
java_mem_args = "-Xmx14g"
```

File: groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars [hunk 5]
```diff
@@ -5,7 +5,7 @@ deploy_to = "staging"
 state_prefix = "env:/staging"
 aws_profile = "staging-eu-west-2"
 cpu_units = 2048
-memory = 16384
+memory = 30720
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars]: Please revert this change.

File: groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars [hunk 6]
```diff
@@ -38,3 +38,5 @@ mongodb_read_preference = "SECONDARY"
 
 results_initial_capacity = 1000000
 results_expiry_time_in_millis = 3600000 # 1 hour
+cache_expiry_in_seconds = 1800
+java_mem_args = "-Xmx14336m"
```
Comment (dlgroves-ch) [inline groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars]: Should be:

```
java_mem_args = "-Xmx14g"
```

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]

File: groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars (hunk 6 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars]: Resolved in latest commit.

File: groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars (hunk 5 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/staging-eu-west-2/staging/vars]: Resolved in latest commit.

File: groups/data-reconciliation/profiles/live-eu-west-2/live/vars (hunk 4 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/live-eu-west-2/live/vars]: Resolved in latest commit.

File: groups/data-reconciliation/profiles/live-eu-west-2/live/vars (hunk 3 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/live-eu-west-2/live/vars]: Resolved in latest commit.

File: groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars (hunk 2 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars]: Resolved in latest commit.

File: groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (kinpang-CH) [reply groups/data-reconciliation/profiles/development-eu-west-2/cidev/vars]: Resolved in latest commit.

**Review by dlgroves-ch — APPROVED** [review]

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
You need to make sure the new variables are passed in in main.tf

**Review by dlgroves-ch — APPROVED** [review]
